docs/general/user-defined-functions.rst

```diff
@@ -197,31 +197,48 @@ Optionally, you can provide a schema::
 Supported languages
 ===================
 
-CrateDB currently only supports the UDF language ``javascript``.
+CrateDB currently only supports the ``JavaScript`` user-defined functions.
 
 .. _udf_lang_js:
 
 JavaScript
 ----------
 
-The UDF language ``javascript`` supports the `ECMAScript 5.1`_ standard.
+The user defined function JavaScript is compatible with the `ECMAScript 2019`_
+specification.
 
 .. NOTE::
 
    The JavaScript language is an :ref:`enterprise feature
    <enterprise-features>`.
 
-CrateDB uses the Java built-in JavaScript engine Nashorn_ to interpret and
-execute functions written in JavaScript. The engine is initialized using the
-``--no-java`` option which basically restricts all access to Java APIs from
-within the JavaScript context. CrateDB's engine also does not allow
-non-standard syntax extensions (``--no-syntax-extensions``).
+CrateDB uses the `GraalVM JavaScript`_ engine as a JavaScript (ECMAScript)
+language execution runtime. The `GraalVM JavaScript`_ engine is a Java
+application that works on the stock Java VMs. The interoperability between
```

Comment:
"VM" should be expanded the first time it is used in a document, with the expanded version put into italics, like this:

> *Virtual Machines* (VMs)